base_control/Motor.ino

```diff
@@ -0,0 +1,110 @@
+Motor::Motor() {}

```

Comment:
Make this file a .cpp instead.
Also, please remember to apply each comment to all the files.